src/main/plugins/org.dita.html5/xsl/simpletable.xsl

```diff
@@ -216,7 +217,17 @@ See the accompanying LICENSE file for applicable license.
   <xsl:template match="*[contains(@class, ' topic/stentry ')]" mode="simpletable:entry">
     <xsl:apply-templates select="." mode="table:common"/>
     <xsl:apply-templates select="." mode="headers"/>
-    <xsl:apply-templates/>
+    <xsl:choose>
+      <xsl:when test="*|text()|processing-instruction()">
+        <xsl:apply-templates/>
+      </xsl:when>
+      <xsl:when test="@specentry">
+        <xsl:apply-templates select="@specentry"/>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:text>&#160;</xsl:text>
```

Comment:
It's probably XSLT 1.0 era, when you had to use output method `xml` for XHTML. Now we have actual `xhtml`, so it will not result in empty elements in XHTML output. My preference is to not output those, unless you really can find a case where modern or semi-modern browsers have issues with the output without the NBSP.